roles/fio_distributed/templates/server_vm.yml.j2

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  name: 'fio-server-{{item | string}}-{{ trunc_uuid }}'
+  namespace: '{{ operator_namespace }}'
+  labels:
+    app: fio-benchmark-{{ trunc_uuid }}
+spec:
+  domain:
+    devices:
```

Comment:
Unless a 1vcpu guest is always sufficient for the fio server, it would be good to add vcpu statements and make it configurable:
spec:
  domain:
    cpu:
      sockets: {{ vcpus }} 

Making cpus configurable is not as important for pods since they will naturally burst and not be constrained by a single vcpu process in the host by default.

Reply:
I've added an option called `vm_cores` that will configure the amount of cores available to the VM... I assume you meant `cores:` and not `sockets:`?

Reply:
It may not matter much for this type of test, but a general recommendation is to use sockets -- that is what "regular" kvm vcpus default to, a 'sockets' vcpu topology enables a guest sched_domain that has a sync wake sched hint that can help some types of workloads that do message passing, context switching, etc.